projects/eslint-plugin/configs/all.json

```diff
@@ -11,6 +11,7 @@
                 "currentProject": "(?<=projects/)([-\\w]+)",
                 "ignoreImports": ["\\?raw", "@taiga-ui/testing/cypress", "@taiga-ui/testing/setup-jest"]
             }
-        ]
+        ],
+        "@taiga-ui/strict-tui-doc-example": "error"
```

Comment:
could you please fix some files in that PR
<img width="784" alt="image" src="https://user-images.githubusercontent.com/12021443/215320737-82e5c335-c3aa-4d09-a53c-08286a9cc579.png">